Tests/TestingTests/TestSupport/TestingAdditions.swift

```diff
@@ -366,3 +366,27 @@ extension Trait where Self == TimeLimitTrait {
     return Self(timeLimit: timeLimit)
   }
 }
+
+extension Issue {
+  init(kind: Kind) {
+    self.init(kind: kind, sourceContext: .init())
+  }
+
+  init(kind: Kind, sourceContext: SourceContext) {
+    self.init(kind: kind, comments: [], sourceContext: sourceContext)
+  }
+}
+
+extension SourceContext {
+  init() {
```

Comment:
If you want to be really concise, I suppose this initializer could just have two default args, yeah?

Reply:
I tried that originally, but the compiler notes that it will cause infinite recursion:

```swift
  init(backtrace: Backtrace? = .current(), sourceLocation: SourceLocation? = nil) {
    self.init(backtrace: backtrace, sourceLocation: sourceLocation) // warning: function call causes an infinite recursion
  }
```

I'm not aware of a way to request that an overload in another module such as this call the _original_ initializer, to resolve that warning.

Reply:
Dang it, alright.